### ISSUE TITLE
is it totally dead?

### ISSUE THREAD

**Issue by sezlony** [PowerCraft/2.0#5]
will there be any update to this mod or is it once and for all dead?

loved it very much in the golden age and would like to play it again, but without the weasel it's not that fun :(

**Comment by DannikInfo** [comment]
not enough time to look into it, but your reaction made me think about looking for it

**Comment by sezlony** [comment]
wow, best developer reaction I've ever got! :D

thanks and fingers crossed for any progress you make!!!

**Comment by Pherex** [comment]
There is desire to see this mod revived.  I miss programming my level 8
miner and watching it remotely.  It was an awesome mod.

**Comment by sezlony** [comment]
any news on this? 😊👍

**Comment by DannikInfo** [comment]
Sorry, but now there is no time :( I want to, but I can not continue to develop this mod yet :(

**Comment by sezlony** [comment]
awww... i had high hopes for this, but i can wait (not forever though) 😃

**Comment by sezlony** [comment]
if only you could make some minor improvements/bugfixes, like:
- harvester block didn't harvest builder block
- laser lenses would be "stackable" again for making higher tier killer lenses

maybe?

**Comment by sezlony** [comment]
hopefully one day everything comes together and the future will bring us the so much waited update 😃

**Comment by sezlony** [comment]
news? 😕

**Comment by sezlony** [comment]
any signs of life here? 😞

**Comment by sezlony** [comment]
up 🤞

**Comment by sezlony** [comment]
upper 👍

**Comment by sezlony** [comment]
uppest 👍

**Comment by sezlony** [comment]
are you taken and died in the war in Ukraine? 😕

**Comment by rapus95** [comment]
@sezlony No. I just wasn't capable of carrying the guilt of letting you down back then. So I tried to dodge the interaction x)

Since the rise of AI for Coding this year, I've spent many hours co-developing a new runtime for possible successors of Weasel with the AI. (Since, to be honest, I ALSO WANT THE MINER BACK!!! 😂) And I'm finally getting somewhere :D
If you're into that science, it'll be a RiscV based runtime. So anything that can compile to bare RiscV could in theory run on the miner. That's the idea at least. Right now, efforts are entirely focused on scriptable stuff, like Miner and Microchip Block.

**Comment by 167OSS** [comment]
### 

> [@sezlony](https://github.com/sezlony)不，我当时实在无法承受让你失望的愧疚感，所以就尽量避免和你见面 x)
> 
> 今年人工智能编程兴起以来，我花了大量时间与人工智能合作开发新的运行时环境，以期为 Weasel 的未来发展方向铺路。（说实话，我也想让挖矿程序回归！！！😂）现在终于取得了一些进展 :D 如果你对这方面感兴趣，它将是一个基于 RISC-V 的运行时环境。理论上，任何能够编译成纯 RISC-V 程序的东西都可以在挖矿程序上运行。至少目前是这样。现在，我们的工作重心完全放在可脚本化的功能上，比如挖矿程序和 Microchip Block。

I hope it still 1.7.10
